src/urbit/app/chess.hoon

```diff
@@ -614,10 +683,53 @@
       ?+  -.sign  (on-agent:default wire sign)
         %fact
           ?+  p.cage.sign  (on-agent:default wire sign)
+             %chess-game
+              :_  this
+              :~  ::  notify us opponent accepted
+                  :*  %pass   /hark-store
+                      %agent  [our.bowl %hark-store]
+                      %poke   :-  %hark-action
+                              !>  :-  %unread-count
+                                  :-  %chess
+                                  :-  /challenges
+                                  :-  %.y  1
+                  ==
+                  :*  =/  title=(list content:hark)
+                        ~[[%text (crip "{<src.bowl>} accepts your challenge")]]
+                      =/  =bin:hark
+                        [/chess [%chess /challenges]]
+                      =/  hark-action=action:hark
+                        ::  XX: should link to game
+                        [%add-note bin title ~ now.bowl / /chess]
+                      =/  =cage
+                        [%hark-action !>(hark-action)]
+                      [%pass /hark-store %agent [our.bowl %hark-store] %poke cage]
+                  ==
+              ==
```

Comment:
The right place to get this to trigger is in the `on-watch` code, since that's how our agent knows that another agent accepted a game: we have an open challenge for them and then they subscribe to us on a new game path. In practical terms, this means lines `533` and `565` of this file as is.